src/app/organizations/state/organization-members.service.ts

```diff
@@ -1,20 +1,35 @@
-import { HttpClient } from '@angular/common/http';
+import { HttpErrorResponse } from '@angular/common/http';
 import { Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material';
 import { ID } from '@datorama/akita';
+import { finalize } from 'rxjs/operators';
 
-import { OrganisationUser } from '../../shared/swagger';
+import { AlertService } from '../../shared/alert/state/alert.service';
+import { OrganisationsService, OrganisationUser } from '../../shared/swagger';
+import { UserQuery } from '../../shared/user/user.query';
 import { OrganizationMembersStore } from './organization-members.store';
+import { OrganizationStore } from './organization.store';
 
 @Injectable({ providedIn: 'root' })
 export class OrganizationMembersService {
 
   constructor(private organizationMembersStore: OrganizationMembersStore,
-              private http: HttpClient) {
+    private organisationsService: OrganisationsService, private userQuery: UserQuery, private organizationStore: OrganizationStore,
+    private alertService: AlertService, private matSnackBar: MatSnackBar) {
   }
 
+  /**
+   * Update the Akita store with the latest array of organization users
+   *
+   * @param {OrganisationUser[]} organizationUsers  The latest array of organization users
+   * @memberof OrganizationMembersService
+   */
```

Comment:
Should the method have the `@transaction()` annotation, given all the store interactions that are happening?